src/components/NotificationPanel/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./styles.css";
+import NotificationBox from "./NotificationBox";
+
+/* window.onload = function() {
+  console.log("darkness");
+  document.getElementById("overlay").style = "display: block";
+};
+
+window.onunload = function() {
+  console.log("hello");
+  document.getElementById("overlay").style = "display: none";
+}; */
+
+const NotificationPanel = ({ notifications }) => (
+  <div>
+    <div className="overlay" />
+    <div className="NotificationPanelBackground" />
+    <div className="NotificationPanel">
+      {Object.keys(notifications).map(item => (
```

Comment:
i thought notifications was an array, so im confused why Object.keys worked

Reply:
i think just notifications.map would be fine, but im not sure

Reply:
notifications.map doesn't work